universal-login-wallet/cypress/integration/createAccount.test.ts

```diff
@@ -9,11 +9,22 @@ describe('E2E: Create Account', () => {
       cy.approveTerms();
       cy.ensureCorrectLocation('/selectDeployName');
       cy.pickUsername(ensName);
-      cy.get('.unilogin-component-waitingfor-action-title');
-      cy.checkLocalStorageTokenProp('wallet-ganache', 'kind', 'Deploying');
-      cy.ensureCorrectLocation('/creationSuccess');
-      cy.goToDashboard();
-      cy.checkLocalStorageTokenProp('wallet-ganache', 'kind', 'Deployed');
+      cy.get('.unilogin-component-suggestions-item-btn').should('not.exist').then(() => {
+        const wallet = JSON.parse(localStorage.getItem('wallet-ganache'));
+        if (wallet.kind === 'Future') {
+          cy.selectTopUpWithCrypto();
+          cy.topUpAccount();
+          cy.checkLocalStorageTokenProp('wallet-ganache', 'kind', 'Future');
+          cy.ensureCorrectLocation('/creationSuccess');
+          cy.checkLocalStorageTokenProp('wallet-ganache', 'kind', 'Deployed');
+        } else {
+          cy.get('.unilogin-component-waitingfor-action-title');
+          cy.checkLocalStorageTokenProp('wallet-ganache', 'kind', 'Deploying');
+          cy.ensureCorrectLocation('/creationSuccess');
+          cy.goToDashboard();
+          cy.checkLocalStorageTokenProp('wallet-ganache', 'kind', 'Deployed');
+        };
```

Comment:
In this flow, it is impossible to have deploying wallet. We have already chosen username, so the wallet state will be `Future`